### ISSUE TITLE
[Rendering] Text block in a container with background image is not accessible

### ISSUE THREAD

**Issue by anindara** [microsoft/AdaptiveCards#6609]
### Target Platforms

NodeJS

### SDK Version

1.3

### Application Name

Viva Connections

### Problem Description

Adding a text block in a container with background image is not accessible. Switching to a light color doesn't help either. Should there be a support for opacity?

### Screenshots

![image](https://user-images.githubusercontent.com/22333452/137967386-2b1d3366-b727-4531-bcab-481802fe2896.png)

### Card JSON

```JSON
{
    "type": "AdaptiveCard",
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.3",
    "body": [
        {
            "type": "Container",
            "backgroundImage": {
                "url": "https://microsoft.sharepoint-df.com/teams/MEEDemo/SiteAssets/Dashboard/tree-small.jpeg"
            },
            "items": [
                {
                    "type": "TextBlock",
                    "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
                    "wrap": true
                }
            ],
            "bleed": true
        }
    ],
    "backgroundImage": {
        "horizontalAlignment": "Center"
    },
    "minHeight": "0px"
}
```
